server/data/projects.js

```diff
@@ -93,7 +97,57 @@ const createProject = async (projectObjParam, user) => {
 	return createdProject;
 };
 
+const updateProject = async (projectObjParam, id, user) => {
+	let projectCheck = await getProjectById(id);
+	ownercheck = checkuseraccess(user, projectCheck.owner);
+	if (!ownercheck)
+		throw forbiddenErr(
+			`Not Authorised to update this project. Not Project Owner`
+		);
+	const projectObj = isValidProjectObject(projectObjParam);
+	const { name, description, github, media, technologies, deploymentLink } =
+		projectObj;
+	const date = new Date();
+	const updateProjectObject = {
+		name,
+		description,
+		github,
+		media,
+		deploymentLink,
+		updatedAt: date,
+		technologies,
+	};
+
+	const projectCollection = await projects();
+	const updateInfo = await projectCollection.updateOne(
+		{ _id: ObjectId(id) },
+		{ $set: updateProjectObject }
```

Comment:
or you could do it here directly like:
```javascript
$set: {
    ...updateProject,
    ...updateProjectObject
}
```

this will take the original project keys, and then overwrite values from the second object while keeping the rest of them from the original object